.rubocop_local.yml

```diff
@@ -0,0 +1,19 @@
+#
+# Overrides
+#
+GlobalVars:
+  AllowedVariables:
+  # Loggers
+  - $log
+  - $vim_log
+#
+# Special Exclusions
+#
+AllCops:
+  TargetRubyVersion: 2.2
```

Comment:
I feel like this belongs in the rubocop_base.yml on ManageIQ/guides if we need it